docs/registry/upload/_index.md

```diff
@@ -279,7 +279,9 @@ For more information, see the [`viam module` command](/fleet/cli/#module)
 
 ### Update an existing module using a GitHub action
 
-To update an existing module in the [Viam registry](https://app.viam.com/registry) using continuous integration (CI), use the [`upload-module` GitHub action](https://github.com/viamrobotics/upload-module).
+To update an existing module in the [Viam registry](https://app.viam.com/registry) using continuous integration (CI), you can use one of two Github actions.
```

Comment:
> update an existing module

'existing module' part matters a lot

if `viam module create` and `viam module update` have not been run, cloud builds will fail (with useful error messages though)

your call whether to emphasize this here, or let users find out

Reply:
Added